src/extwchar/unw16ifexc.h

```diff
@@ -41,8 +41,10 @@ bool isExclusion (wint_t uv)
 	|| (64320 <= uv && uv <= 64321)
 	|| (64323 <= uv && uv <= 64324)
 	|| (64326 <= uv && uv <= 64334)
+  #if SIZEOF_WCHAR_T > 2
```

Comment:
Really? wint_t != wchar_t
wint_t is always 4 bytes, wchar_t 2 or 4.

Reply:
This was my key misunderstanding. I haven't worked with `wint_t` before and read that it could hold a `wchar_t` so I assumed they were the same size. I'll revert the changes related to that.